docs/app/documentation/component-docs/time-picker/examples/time-picker-other-delimiter-example.component.ts

```diff
@@ -0,0 +1,148 @@
+import { Component, Injectable } from '@angular/core';
+import { TimeFormatParser } from '../../../../../../library/src/lib/time-picker/format/time-parser';
+import { TimeObject } from '../../../../../../library/src/lib/time/time-object';
+
+
+@Injectable()
+export class TimeFormatDashes extends TimeFormatParser {
```

Comment:
Rename the class since this isn't dashes anymore - maybe something like 'CustomTimeFormat'?

Otherwise this PR looks pretty good